### PR TITLE
Use team slug instead of OrgId.

### LINKED ISSUE #198: All spaces currently showing up for all teams.
All calls to /api/space/list have the same orgID of "ADNO".
SpaceContainer, Ln 39: 
const spaces = await this.context.fetch(`/api/${this.context.team.slug}/spaces/list?orgId=ADNO`);

### PR DESCRIPTION
**Pull request by jcubbage**
Fixes #198 
Only uses orgcode, ignores chart. Not sure if that's an issue. At least in dev, no chart in space table.

### CODE REVIEW

**Comment by jSylvestre** [comment]
@jcubbage Since you are changing the SpacesController, do you want to change the !String.Is... to !string.Is... (lower case s)

File: Keas.Mvc/Controllers/Api/SpacesController.cs [hunk 1]
```diff
@@ -31,11 +31,12 @@ public async Task<IActionResult> SearchSpaces(string q)
             return Json(space);
         }
 
-        public async Task<IActionResult> List(string orgId)
+        public async Task<IActionResult> List(string teamSlug)
         {
             //TODO clean up workstations query
+            var orgIds = await _context.FISOrgs.Where(f => f.Team.Slug == teamSlug).Select(x => x.OrgCode).ToListAsync();
```
Comment (jSylvestre) [inline Keas.Mvc/Controllers/Api/SpacesController.cs]: distinct

**Comment by jSylvestre** [comment]
`namespace Keas.Core.Domain
{
    // Facilities link deptSpace, flattened
    // ties together rooms and departments
    [Table("Space")]
    public class Space {
`


image

File: Keas.Mvc/ClientApp/components/Spaces/SpacesContainer.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default class SpacesContainer extends React.Component<{}, IState> {
     }
 
     public async componentDidMount() {
-        const spaces = await this.context.fetch(`/api/${this.context.team.slug}/spaces/list?orgId=ADNO`);
+        const spaces = await this.context.fetch(`/api/${this.context.team.slug}/spaces/list?teamSlug=${this.context.team.slug}`);
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/Spaces/SpacesContainer.tsx]: @jcubbage the slug is already in the URL why would you pass in twice or do I not want to know?

File: Keas.Mvc/Controllers/Api/SpacesController.cs [hunk 3]
```diff
@@ -24,44 +23,44 @@ public async Task<IActionResult> SearchSpaces(string q)
...
-                || (!String.IsNullOrWhiteSpace(x.RoomNumber) && queryWords.Any(s => x.RoomNumber.ToLower().Contains(s))))
+                .Where(x => (!string.IsNullOrWhiteSpace(x.BldgName) && queryWords.Any(s => x.BldgName.ToLower().Contains(s)))
+                || (!string.IsNullOrWhiteSpace(x.RoomName) && queryWords.Any(s => x.RoomName.ToLower().Contains(s)))
+                || (!string.IsNullOrWhiteSpace(x.RoomNumber) && queryWords.Any(s => x.RoomNumber.ToLower().Contains(s))))
                 .AsNoTracking().ToListAsync();
             return Json(space);
         }
 
-        public async Task<IActionResult> List(string orgId)
+        public async Task<IActionResult> List(string teamSlug)
```
Comment (srkirkland) [inline Keas.Mvc/Controllers/Api/SpacesController.cs]: @jcubbage if you just take team name as a param here it should map properly, and then you can get rid of that teamSlug query string param you added

File: Keas.Core/Domain/Space.cs [hunk 4]
```diff
@@ -5,6 +5,7 @@ namespace Keas.Core.Domain
 {
     // Facilities link deptSpace, flattened
     // ties together rooms and departments
+    [Table("Spaces")]
```
Comment (srkirkland) [inline Keas.Core/Domain/Space.cs]: @jSylvestre real table is singular now right?

Comment (jSylvestre) [reply Keas.Core/Domain/Space.cs]: @srkirkland Correct. Table with the data is "Space"

Comment (jcubbage) [reply Keas.Core/Domain/Space.cs]: That's what I put!

**Pull request merged**